frontend/src/components/DatabaseBackup/DatabaseBackupSettingForm.vue

```diff
@@ -0,0 +1,413 @@
+    <div class="py-0.5 space-y-6">
+      <div class="flex flex-col gap-y-2">
+        <label class="textlabel">
+          {{ $t("database.backup-setting.form.schedule") }}
+        </label>
+        <NPopover
+          trigger="manual"
+          :show="state.showBackupPolicyEnforcement"
+          :show-arrow="false"
+          placement="bottom-start"
+        >
+          <template #trigger>
+            <div class="flex items-center gap-x-2 text-sm">
+              <label
+                v-for="schedule in PLAN_SCHEDULES"
+                :key="schedule"
+                class="flex items-center gap-x-2"
+                :class="
+                  !isAllowedScheduleByPolicy(schedule) &&
+                  'opacity-50 cursor-not-allowed'
+                "
+                @click="setSchedule(schedule)"
+              >
+                <input
+                  type="radio"
+                  :value="schedule"
+                  :checked="schedule === checkedSchedule"
+                  :disabled="!isAllowedScheduleByPolicy(schedule)"
+                />
+                <span>{{ nameOfSchedule(schedule) }}</span>
+              </label>
+            </div>
+          </template>
+
+          <router-link
+            class="normal-link text-sm"
+            :to="`/environment/${database.instance.environment.id}`"
+          >
+            {{
+              $t("database.backuppolicy-backup-enforced-and-cant-be-disabled", [
+                $t(`database.backup-policy.${backupPolicy}`),
+              ])
+            }}
+          </router-link>
+        </NPopover>
+      </div>
+
+      <div v-if="checkedSchedule === 'WEEKLY'" class="flex flex-col gap-y-2">
+        <label class="textlabel">
+          {{ $t("database.backup-setting.form.day-of-week") }}
+        </label>
+        <div class="w-[16rem]">
+          <BBSelect
+            :selected-item="
+              localFromUTC(state.setting.hour, state.setting.dayOfWeek)
+                .dayOfWeek
+            "
+            :item-list="AVAILABLE_DAYS_OF_WEEK"
+            @select-item="setDayOfWeek"
+          >
+            <template #menuItem="{ item: day }">
+              {{ nameOfDay(day) }}
+            </template>
+          </BBSelect>
+        </div>
+      </div>
+
+      <div
+        v-if="checkedSchedule === 'WEEKLY' || checkedSchedule === 'DAILY'"
+        class="flex flex-col gap-y-2"
+      >
+        <label class="textlabel">
+          <span>
+            {{ $t("database.backup-setting.form.time-of-day") }}
+          </span>
+          <span class="ml-1 textinfolabel">
+            ({{ Intl.DateTimeFormat().resolvedOptions().timeZone }})
+          </span>
+        </label>
+        <div class="w-[16rem]">
+          <BBSelect
+            :selected-item="
+              localFromUTC(state.setting.hour, state.setting.dayOfWeek).hour
+            "
+            :item-list="AVAILABLE_HOURS_OF_DAY"
+            @select-item="setHour"
+          >
```

Comment:
This may be a paid feature? @tianzhou @candy2255

Reply:
Might provide a crontab syntax based backup schedule setting for enterprise lol, wdyt?

Reply:
SGTM.